source/release.rst

```diff
@@ -143,17 +136,26 @@ Or:
         git fetch origin master
         git checkout -b <branch-name> origin/master
 
-3. Run the command below. The ``conda/`` folder is where ``meta.yaml`` is located and the ``-c``
-tells conda the channels where the dependencies defined in ``meta.yaml`` can be found.
+3. Run ``conda env list``. There will be several paths listed.
+For each of those paths, run ``rm -rf <path>/miniconda3/conda-bld`` to ensure that you're building the latest package:
```

Comment:
This is not necessary.  You only need to delete the `conda-bld` environment associated with your `miniconda3` installation.  There isn't one per environment.  The only reason you might see this in your particular case is because you have an installation on `anaconda3` and of `miniconda3`, which are completely separate from one another.

```suggestion
3. Run ``conda env list``. Determine the path for the `miniconda3` installation you are using to build the package. 
Typically, this will be `~/miniconda3`.  Run ``rm -rf <path>/conda-bld`` to ensure that previously built packages are not included in the current build.
```